### ISSUE TITLE
add another game to the library

### ISSUE THREAD

**Issue by wolffles** [wolffles/game-iverse#2]
if you enjoyed the brick break game and would like to try your hand at supplying building blocks for other games, don't be afraid to try it.

**Comment by Roodiger** [comment]
I can't get the brickbreaker game to work. It is telling me:

brickbreakclasses.js:4 Uncaught TypeError: Cannot read property 'getContext' of null
    at new World (brickbreakclasses.js:4)
    at example.js:4

**Comment by Roodiger** [comment]
I gave it a shot, but I'm new to this whole GitHub open-source thing. I don't think I know how it works quite yet so I removed my pull request. I don't think I have understanding of NPM and everything to successfully contribute what you need unfortunately. It seems like a good idea though, I love what you're doing.

**Comment by wolffles** [comment]
Hey man don't beat yourself up, your code was amazing I played the game and
really liked it much better than the brick break game! The project itself
is a stretch because node packages aren't supported by browsers yet so even
if we install the package on node, we can use it client-side... yet.

I have your code on my local repo now I can convert it to the package
format and I'll let you know when Its done. Keep on trying man you're an
excellent coder.

**Comment by Roodiger** [comment]
Yeah let me know! I'm interested to see how it looks converted into the proper format. 

I added the HTML and stuff just for my own testing and such. 

Just let me know when you have something that works! Im going to look more into node packages.

**Comment by modaserMoj** [comment]
Is this still active? I was thinking of adding rock, paper, scissors?

**Comment by ghmarsh** [comment]
Is this still active? I was thinking of adding something.

**Comment by wolffles** [comment]
Oh man, I started this project for a school project. I don’t maintain it anymore. Wish I had the time and motivation these days but don’t.

Wolfgang
